pages/osx/nettop.md

```diff
@@ -0,0 +1,35 @@
+# nettop
+
+> Display dynamic real-time information about the network.
+
+- Monitor TCP and UDP sockets from all interfaces:
+
+`nettop`
+
+- Monitor TCP sockets from Loopback interfaces:
+
+`nettop -m {{tcp}} -t {{loopback}}`
+
+- Monitor a specific process:
+
+`nettop -p {{process_id|process_name}}`
+
+- Display summary only for per-process:
+
+`nettop -P`
+
+- Monitor in logging mode and display 10 samples:
```

Comment:
From how I understood it in the man page, this command doesn't run periodically right? To avoid using `logging mode`, how about
```suggestion
- Print 10 samples of real-time network information: 
```

Reply:
Yes, this command run only one time, print samples periodically, default Interval time is 1 second. I don't know if this can be called `real-time` 😂

Reply:
I jut took this part from the command description:

> Display dynamic real-time information about the network.

But we can of course just remove the `real-time`.

Reply:
Oh, it's my fault... I copy the words `dynamic` and `real-time` from `top` command to description, my apologies.

Maybe we should use the command description from `man nettop`: 
> Display updated information about the network

Reply:
So, maybe `Print 10 samples of network information` ?

Reply:
OK, I have applied `Print 10 samples of network information`, 
and change command description to `Display updated information about the network`